src/test/rgw/test_multi.py

```diff
@@ -347,21 +347,25 @@ def init(parse_args):
                     arg = ['--display-name', '"Test User"']
                     arg += user_creds.credential_args()
                     if args.tenant:
-                        cmd += ['--tenant', args.tenant]
+                        arg += ['--tenant', args.tenant]
                     user.create(zone, arg)
                 else:
                     # read users and update keys
                     admin_user.info(zone)
                     admin_creds = admin_user.credentials[0]
-                    user.info(zone)
+                    arg = []
+                    if args.tenant:
+                        arg += ['--tenant', args.tenant]
+                    user.info(zone, arg)
```

Comment:
i feel like we'd be better off extending `class User` in `rgw_multi/multisite.py` to take a tenant on construction, ie
`235: user = multisite.User('tester', tenant=args.tenant)`
then its `user_arg()` function could add this `--tentant` option to all of its operations, and your `get_tenant()` function could return user.tenant instead of stashing it in Config

Reply:
thanks @cbodley will do the test fixes in a separate small PR